searx/search/processors/online.py

```diff
@@ -9,6 +9,8 @@
 from timeit import default_timer
 import asyncio
 import ssl
+from typing import Dict, List
+
 import httpx
 
 import searx.network
```

Comment:
@dalf In context of https://github.com/searxng/searxng/issues/762 I have the question: do we need this `import searx.network` any longer or can we now import by:

```python
from searx.network import get, post
```

and by example in line 110 ff

```python
        if params['method'] == 'GET':
            req = get
        else:
            req = post
```

No need to change, just a question for my knowledge

Reply:
This PR does not fix #762 because there is still a conflict between the global variable name `network` defined by `settings.yml` and the `network` module.

The issue exists only the engine modules and for `online.py`. These lines are not an issue because `searx.network` is going to be override:
https://github.com/searxng/searxng/blob/1878f67e642dc6a640c0b3a72fe5c6e060bcbc81/searx/search/processors/online.py#L16
https://github.com/searxng/searxng/blob/1878f67e642dc6a640c0b3a72fe5c6e060bcbc81/searx/search/processors/online.py#L107-L110

---

IMO, the issue #762 should be solved in another way, but that's topic of another discussion.